compiler/ballerina-lang/src/main/resources/ballerina-toml-schema.json

```diff
@@ -155,7 +155,7 @@
       "properties": {
         "java11": {
           "type": "object",
-          "additionalProperties": false,
+          "additionalProperties": true,
```

Comment:
Why do we need to allow additional properties ?

Reply:
This is not needed for java11.